pages/price-feeds/create-your-first-pyth-app/evm/part-2.mdx

```diff
@@ -246,13 +257,18 @@ async function run() {
 run();
 ```
 
-Notice that the first half of this file defines a variable called `abi`.
-This variable represents the interface of our contract and can be autogenerated by running `forge inspect MyFirstPythContract abi` in the `contracts` directory.
-The `run` function does two different things.
-First, it instantiates an interface to our contract using the `viem` library.
-Second, it uses `EvmPriceServiceConnection` from `@pythnetwork/pyth-evm-js` to retrieve
+This script defines the `abi` which represents the interface of our contract and can be autogenerated by running `forge inspect MyFirstPythContract abi` in the `contracts` directory.
+
+The `run` function does two different things:
+
+- First, it instantiates an interface to our contract using the [`viem`](https://viem.sh/) library.
+- Second, it uses `EvmPriceServiceConnection` from [`@pythnetwork/pyth-evm-js`](<(https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/js)>) to retrieve
+
+Run this command using
```

Comment:
hmm suggest "Run this script using:" ? It's not really a command. (Yes i realize I wrote that before but it does seem odd now)

(The ending colon is required to introduce blocks of content afterward. Notice that we do that for all of the other content blocks above)